mage_ai/frontend/pages/pipelines/[pipeline]/runs/[run]/index.tsx

```diff
@@ -65,7 +72,9 @@ function PipelineBlockRuns({
 
   const { data: dataPipelineRun } = api.pipeline_runs.detail(
     pipelineRunProp.id,
-    {},
+    {
```

Comment:
We still need to periodically fetch the pipeline run so that we can display its running status, as well as its `execution_date` in the breadcrumbs:
![image](https://github.com/mage-ai/mage-ai/assets/78053898/38df382d-ebce-4796-b267-8baa5650eb5d)